### ISSUE TITLE
Total estimated value doesn't include "before cargo" list in calculation (new run creation page)

### ISSUE THREAD

**Issue by Seprum** [molbal/abyss-tracker#98]
**Describe the bug**
When I use "My cargo is not empty!" option then total loot value doesn't exclude items specified in "Before cargo" input

**To Reproduce**
1. Go to 'Add run'
2. Click on 'My cargo is not empty!'
3. Paste some loot in both textareas
4. Check "Total value is approximately"

**Expected behavior**
"Before cargo" list's total value should be excluded from "After cargo" list's value.

**Additional context**
Honestly, I was planning to fix it by myself, but it's easier to farm my next Omega in abyss than to figure out how to set up a working dev environment with all that stuff project is currently built with... Gave up after fixing several "composer update" errors due to package versions incompatibility (not a php dev myself though).

**Comment by molbal** [comment]
Hey,

It's been a long bug thats relatively easy to fix - I agree, that a dev environment is painful to set up. I shall make a guide for contributors and make a DB that you can import without _opsec_ data in it.

+ the project uses [Laravel nova](https://nova.laravel.com/) to make my life easier, but I purchased a license for it, which does not let me share that package. (That's why I put that in a separate private repo)

Long story short, I'm working on an opsec patch now and I'll fix this bug after that's online. (After 2021.06.07)
